spec/task_005_lib_spec.rb

```diff
@@ -0,0 +1,184 @@
+require './lib/task_005_lib'
+
+RSpec.describe Task005 do
+  describe 'parse_input(input_data)' do
+    it ' parses input data ' do
+      input = <<~INPUT
+        0,9 -> 5,9
+        8,0 -> 0,8
+        9,4 -> 3,4
+      INPUT
+
+      lines = Task005.parse_input(input)
+
+      expect(lines).to contain_exactly(
+        have_attributes(
+          point1: have_attributes(x: 0, y: 9),
+          point2: have_attributes(x: 5, y: 9),
+        ),
+        have_attributes(
+          point1: have_attributes(x: 8, y: 0),
+          point2: have_attributes(x: 0, y: 8),
+        ),
+        have_attributes(
+          point1: have_attributes(x: 9, y: 4),
+          point2: have_attributes(x: 3, y: 4),
+        ),
+      )
+    end
+  end
+end
+
+RSpec.describe Task005::Line do
+  describe 'horizontal_or_vertical?' do
```

Comment:
Don't you want to follow convention and mark instance methods with `#`?

```suggestion
  describe '#horizontal_or_vertical?' do
```